src/scopes/index.js

```diff
@@ -1,13 +1,38 @@
+/* eslint-disable dot-notation */
+import { _ } from 'lodash';
 import { activityReportsFiltersToScopes as activityReport } from './activityReport';
 import { grantsFiltersToScopes as grant } from './grants';
 import { goalsFiltersToScopes as goal } from './goals';
+import { AWS_ELASTIC_SEARCH_INDEXES, DECIMAL_BASE } from '../constants';
+import { search } from '../lib/awsElasticSearch/index';
 
 const models = {
   activityReport,
   grant,
   goal,
 };
 
+async function checkForSearchItems(filters) {
```

Comment:
Putting this here for now might make sense in the future as this grows to rip it out into its own file.